pages/homepage.js

```diff
@@ -0,0 +1,274 @@
+import { useState } from "react";
+import { useRouter } from "next/router";
+import Calendar from "react-calendar";
+import styled from "styled-components";
+import "react-calendar/dist/Calendar.css";
+
+export default function Home({ data, activities, onDeleteActivity }) {
+  const router = useRouter();
+  const weatherData = data.forecast.forecastday;
+  const [checkedActivities, setCheckedActivities] = useState(false);
+  const [weekDay, setWeekDay] = useState(
+    new Date().toLocaleDateString("default", { year: "numeric" }) +
+      "-" +
+      new Date().toLocaleDateString("default", { month: "2-digit" }) +
+      "-" +
+      new Date().toLocaleDateString("default", { day: "2-digit" })
+  );
+
+  const [showCalendar, setShowCalendar] = useState(false);
```

Comment:
Was thinking that the name might be misleading. But I am not sure. Though that `isCalenderVisiable` might be a better variable name. What do you think?

Reply:
yeah sounds like a good adjustment, i will change it. :)